app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Users
+  # override devise sessions controller to create a json endpoint.
+  class SessionsController < Devise::SessionsController
+    skip_before_action :verify_authenticity_token, only: [:create]
+    respond_to :json
+
+    # POST /resource/sign_in
+    def create
+      user_password = params[:session][:password]
+      user_email = params[:session][:email]
```

Comment:
create strong parameters method to handle this http://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html